[docs] Remove `authToken` from Sentry configuration guide (#21716)
expo/expo: docs/pages/guides/using-sentry.mdx

}
```

> To upload the source map to Sentry, you need to create a [Sentry auth token](https://docs.sentry.io/product/cli/configuration/). After creating your own Sentry auth token [here](https://sentry.io/settings/account/api/), you can configure this token through the `SENTRY_AUTH_TOKEN` environment variable in [EAS Build](../build-reference/variables.mdx#using-secrets-in-environment-variables).

Reviewer: ```suggestion
To upload the source map to Sentry, you must create a [Sentry auth token](https://docs.sentry.io/product/cli/configuration/). After creating your Sentry auth token [here](https://sentry.io/settings/account/api/), you can configure this token through the `SENTRY_AUTH_TOKEN` environment variable in [EAS Build](/build-reference/variables/#using-secrets-in-environment-variables).
```

}
```

To upload the source map to Sentry, you must create a [Sentry auth token](https://docs.sentry.io/product/cli/configuration/). After creating your Sentry auth token [here](https://sentry.io/settings/account/api/), you can configure this token through the `SENTRY_AUTH_TOKEN` environment variable in [EAS Build](/build-reference/variables/#using-secrets-in-environment-variables).

> <Callout>

Author: ```suggestion
<Callout type="warning">
```

Reviewer 2: If you want to use the type, we have implemented similar mechanism to GH, so it will be:
```
> **warning** CONTENT
```

> <Callout type="warning">

Reviewer: Did the following locally
```suggestion
> **warning** Never commit secrets to your repository. Keep the Sentry auth token in a safe place, outside your repository and use the environment variable instead.
```

And it worked for me: 

<img width="860" alt="CleanShot 2023-03-15 at 17 48 15@2x" src="https://user-images.githubusercontent.com/10234615/225306785-4e04e64b-0c4e-4647-a8b0-6beafd7af275.png">

Reviewer: Can you make sure the dependencies are up to date for your local setup?

Reviewer: Yeah, all Callouts need to follow the syntax `**callout-type**`

Reviewer: I will also update our docs/readme.md on how to use different callouts. Just find out that we don't have any info on that.

Never commit secrets to your repository. Keep the Sentry auth token in a safe place, outside your repository and use the environment variable instead.

> </Callout>

Reviewer 2: Instead of importing a component you can just use quote (`>`) Markdown syntax, which is translated to Callout.

Author: I tried, but I couldn't get the callout to be a warning callout :C
